docs/databricks.md

```diff
@@ -71,31 +67,32 @@ Start A Databricks Cluster
 2. Disable autoscaling.
 3. Choose the number of workers that matches the number of GPUs you want to use.
 4. Select a worker type that has a GPU for the worker.
-5. Update the cluster Configuration "Advanced Options" to include an "Init Scripts". Add your init.sh script you uploaded above: "dbfs:/databricks/scripts/init.sh"
+5. Update the cluster configuration "Advanced Options" to include an "Init Scripts". Add your init.sh script you uploaded above: "dbfs:/databricks/scripts/init.sh".
 6. Optionally add other configurations.
 
 Import the GPU Mortgage Example Notebook
 ---------------------------
-1. See https://docs.databricks.com/user-guide/notebooks/notebook-manage.html on how to import a notebook.
-2. Import the notebook: https://github.com/rapidsai/spark-examples/tree/master/notebook/databricks/xgboost_notebook_mortgage-gpu_scala.scala
+1. See [Managing Notebooks](https://docs.databricks.com/user-guide/notebooks/notebook-manage.html) on how to import a notebook.
+2. Import the notebook: [XGBoost4j-Spark mortgage notebook](https://github.com/rapidsai/spark-examples/tree/master/notebook/databricks/mortgage-gpu.scala)
```

Comment:
currently this link points to the master branch, we probably want all links relative to the current branch (so just remove upto .../master/